qiita_db/support_files/patches/18.sql

```diff
@@ -0,0 +1,3 @@
+-- March 18, 2015
+-- Add column to analysis table to mark shopping cart
+ALTER TABLE qiita.analysis ADD def bool NOT NULL DEFAULT false;
```

Comment:
I know you can't use "default" as a column name, but def is confusing. Can you use `dflt` as that is one of the standard default acronyms in CS?